x-pack/plugins/security_solution/server/request_context_factory.ts

```diff
@@ -111,6 +114,15 @@ export class RequestContextFactory implements IRequestContextFactory {
 
       getAuditLogger,
 
+      getRulesManagementClient: (ml?: SetupPlugins['ml']) =>
+        createRulesManagementClient(
+          startPlugins.alerting.getRulesClientWithRequest(request),
+          request,
+          coreContext.savedObjects.client,
+          licensing,
+          ml
```

Comment:
If I understand it correctly, we can use `plugins.ml` here to avoid passing the `ml` parameter via route.

Reply:
Yes, we can! Thanks, this simplifies the code a bunch